app/controllers/hyrax/batch_ingest/batches_controller.rb

```diff
@@ -21,11 +22,9 @@ def create
         # TODO: Is the original_filename is what we really want to put
         # in source_location?
         @batch.source_location = params['batch']['batch_source'].original_filename
-        # TODO: status should only be set to 'accepted' after validation of all
-        # other fields succeeds.
-        @batch.status = 'accepted'
-        # TODO: Use BatchRunner to kick off the ingest process.
-        if @batch.save
+        @batch.status = 'received'
+
+        if start_batch_runner(@batch)
```

Comment:
Should this check be `@batch.valid?` instead?  Or maybe this is good and inside it `runner.read` and `runner.enqueue` should be called before the redirect.  (This might be the same as my comment in `start_batch_runner`.)